src/Database/Query.php

```diff
@@ -327,7 +327,7 @@ public function sql(?ValueBinder $generator = null): string
      * @param callable $visitor A function or callable to be executed for each part
      * @return $this
      */
-    public function traverse($visitor)
+    public function traverse(Closure $visitor)
```

Comment:
This is intentional parameter type widening.

Reply:
Did this throw a warning too in PHP74? Otherwise we want to keep this probably.

Reply:
I know, mentioned in my last comment. But the interface wasn't updated accordingly and doing it now wouldn't be BC?